haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="https://github.com/grahams/alivejournal"
+COPYRIGHT="
+	2000 Simon Huet
+	2010 Sean Graham
+	"
+LICENSE="GNU GPL v2"
+REVISION="1"
+gitSrcRevision="11a4bed0a0800f66ce5069d3df6a0a3cc2be43aa"
+SOURCE_URI="https://github.com/grahams/alivejournal/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="198950a11a0cb38cd9093d08619991ad6bbd2d376e7b1999b09a8a556d9add68"
+SOURCE_FILENAME="alivejournal-$portVersion-$gitSrcRevision.tar.gz"
+SOURCE_DIR="alivejournal-$gitSrcRevision"
+PATCHES="alivejournal-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	alivejournal = $portVersion
+	app:AliveJournal = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	install -d $appsDir $docDir
+	install -t $appsDir objects/AliveJournal
+	addAppDeskbarSymlink $appsDir/AliveJournal
+	install -t $docDir README
+	sed -n '/http://www.bebits.com/app/1920/q;p' $docDir/README
+	addResourcesToBinaries $portDir/additional-files/alivejournal.rdef $appsDir/AliveJournal
```

Comment:
When it's possible to keep the lines <= 80 chars (without having to do uggly things) we prefer to do so.
```diff
-	addResourcesToBinaries $portDir/additional-files/alivejournal.rdef $appsDir/AliveJournal
+	addResourcesToBinaries $portDir/additional-files/alivejournal.rdef \
+		$appsDir/AliveJournal
```

Hmm, did you create the rdef file? If yes, you should "git add" (and "git commit") it.
(You need to "cd haiku-apps/alivejournal && mkdir additional-files" then put the alivejournal.rdef file there.)

Well, if know how to change the upstream Makefile to have it use the rdef file, you could open a PR on the upstream site with all these changes, so that you won't even need to add the rdef file here.
That said, both methods are ok. And I guess we have more recipes with rdef files in our repo than recipes of programs providing their own rdef file.